README.md

```diff
@@ -28,7 +28,18 @@ python3 setup.py install
 
 ## Quick Usage
 
-Use in most commands using the `--network` option:
+First, make sure you have one of the expected environment variables set.
```

Comment:
Does it only require one and not both?

Reply:
I add a bit more to hopefully make it clearer